nixos/modules/system/boot/uki.nix

```diff
@@ -27,6 +27,18 @@ in
         description = lib.mdDoc "Version of the image or generation the UKI belongs to";
       };
 
+      tries = lib.mkOption {
+        type = lib.types.nullOr lib.types.ints.unsigned;
+        default = null;
+        description = lib.mdDoc ''
+          Number of boot attempts before this UKI is considered bad.
```

Comment:
We should probably add a sentence on what happens if this is set to `null`, i.e. that boot counting is disabled.

Reply:
done :)